src/app/shop/[category]/[productId]/_components/Product/OptionWithButtons.tsx

```diff
@@ -76,6 +84,9 @@ export default function OptionWithButton({ productData }: OptionWithButtonProps)
   };
 
   const { mutate: addCartProduct } = useMutation({ mutationFn: (data: CartProductType) => postCart(data) });
+  const { mutate: createOrder } = useMutation({
+    mutationFn: postCreateOrder,
+  });
```

Comment:
```suggestion
const { mutate: addProductToCartMutate } = useMutation({
  mutationFn: (data: CartProductType) => postCart(data),
});

const { mutate: createOrderMutate } = useMutation({
  mutationFn: postCreateOrder,
});
```
굳이 싶긴한데 이런식의 목적과 동작을 더 명확하게 작성하면 팀원들이랑 일관성을 유지하기도 좋고 동떨어진 코드를 확인해도 어떤 동작인지 추론 가능할거같습니다